src/java.desktop/macosx/classes/sun/lwawt/macosx/CPlatformWindow.java

```diff
@@ -321,6 +322,7 @@ public void initialize(Window _target, LWWindowPeer _peer, PlatformWindow _owner
 
         responder = createPlatformResponder();
         contentView.initialize(peer, responder);
+        setAllowAutomaticWindowTabbing();
```

Comment:
This is in the initialize() method that is being called for every window.
That means the value could be changed by the app (back and forth!) during the execution of the program.
We don't want that. So add it in a static initializer block for the class, so it is only ever read ONCE and guaranteed to be read before the first window is created. This then means the Java method isn't needed and the native method will be static.

Reply:
Yeah, i have changed the method call to the native method in  a static block and added the doPrivileged around getProperty.